.github/workflows/certora.yml

```
@@ -13,20 +13,35 @@ on:
   workflow_dispatch:
 
 jobs:
-  verify_stkBNB:
+  get-diff:
     runs-on: ubuntu-latest
-
+    # Map a step output to a job output
+    outputs:
+      output1: ${{ steps.certora.outputs.run }}
+    
     steps:
       - uses: actions/checkout@v3
 
       - name: Get Diff Action
         uses: technote-space/get-diff-action@v4.0.1
-        id: git-diff # via outputs
+        id: get-diff-action # via outputs
         with:
           PATTERNS: |
             +(contracts)/*.sol
             +(contracts|certora)/**/*.sol
 
+      - id: certora
+        run: echo "::set-output name=run::true"
+        if: steps.get-diff-action.outputs.diff
+
+  verify_stkBNB:
+    runs-on: ubuntu-latest
+    if: needs.get-diff.outputs.output1 == 'true'
```

```suggestion
    if: needs.get-diff.outputs.output1
```
Just this should work as the export is a boolean, I suppose?

---

Updated code, not requried booleans now.